### ISSUE TITLE
Error when installing apex, really need your help!

### ISSUE THREAD

**Issue by stayhungry1** [yuexy/PS-ViT#9]
Hello! I installed under the instructions (PyTorch==1.7.1 and torchvision==0.8.2 with CUDA==10.1). When I install apex using "pip install -v --disable-pip-version-check --no-cache-dir --global-option="--cpp_ext" --global-option="--cuda_ext" ./", it successfully finished. However, when I run the code, there is an error from "import apex", that is  "fused_weight_gradient_mlp_cuda module not found. gradient accumulation fusion with weight gradient computation disabled." Have anyone met this problem? Thank you very much!
